.github/workflows/coverity-scan.yml

```
@@ -0,0 +1,55 @@
+name: coverity-scan
+
+# Controls when the action will run.
+on:
+  # Run this action on a schedule (we're allowed a maximum of two per day)
+  schedule:
+    - cron: '0 18 * * SUN' # Sunday at 18:00 UTC
+
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+# A workflow run is made up of one or more jobs that can run sequentially or in parallel
+jobs:
+  build:
+    runs-on: ubuntu-24.04
+    if: github.repository == 'lutraconsulting/MDAL'
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Install Libraries
+        run: |
+          sudo apt-get update
+          sudo apt-get install -y --allow-unauthenticated libgdal-dev
+          sudo apt-get install -y libhdf5-dev libnetcdf-dev
+          sudo apt-get install -y libxml2-dev
+      - name: Configure MDAL
+        run: |
+          cmake . \
+            -DCMAKE_BUILD_TYPE=Debug \
+            -DENABLE_TESTS=ON
+      - name: Download Coverity Build Tool
+        run: |
+          wget -q https://scan.coverity.com/download/linux64 --post-data "token=$TOKEN&project=lutraconsulting%2FMDAL" -O cov-analysis-linux64.tar.gz
+          mkdir cov-analysis-linux64
+          tar xzf cov-analysis-linux64.tar.gz --strip 1 -C cov-analysis-linux64
+        env:
+          TOKEN: ${{ secrets.COVERITY_SCAN_TOKEN }}
+      - name: Build with cov-build
+        run: |
+          pwd
+          export PATH=`pwd`/cov-analysis-linux64/bin:$PATH
+          cov-build --dir cov-int make -j$(nproc)
+      - name: Submit to Coverity Scan
+        run: |
+          tar czvf mdal.tgz cov-int
+          curl \
+            --form project=qgis%2Fo2 \
```

Should this not be:
```suggestion
            --form project=lutraconsulting%2FMDAL \
```
?

---

Oops, yep. Strange that it worked anyway 🤔